core/bentley/src/Logger.ts

```diff
@@ -286,7 +286,12 @@ export class Logger {
    */
   public static logException(category: string, err: any, log: LogFunction = (_category, message, metaData) => Logger.logError(_category, message, metaData)): void {
```

Comment:
The comment for this method should be updated to say "For legacy BentleyError exceptions, the special "exceptionType" property will be added as metadata. Otherwise all enumerable members of the exception are logged as metadata."

Reply:
updated.